Order Page View.md

```diff
@@ -0,0 +1,338 @@
+
+Some new specifications for the new design are in progress but to avoid any features regressions. We need to specify the whole behaviour of this page to make it easier to compare the new scenario and avoid any new bugs and regressions.
+
+**Related issue**:
+
+[https://docs.google.com/spreadsheets/d/1wfwQRJ28cXQmhGDZcW05RpD12HZ2Em3_bVvR1r9_m_k/edit?usp=sharing](https://docs.google.com/spreadsheets/d/1wfwQRJ28cXQmhGDZcW05RpD12HZ2Em3_bVvR1r9_m_k/edit?usp=sharing)
+
+**WHAT**
+
+Thus, this project aims at specifying the new behaviours for every scenario a merchant could be dealing with the order management. This specification should specifically define :
+
+-   Order (order status, invoices, cancellation)
+-   Customer (customer card and addresses)
+-   Paiement
+-   Messages
+-   Refund
+-   Product & voucher
+
+**SPECIFICATIONS**
+
+**Orders > Orders > View order**
+
+The user access to the order view page by either clicking on **the order’s row** or **the view button**.
+
+**Generic page operation - Order view**
+
+On the header, we have **the order’s reference, customer’s name, total price in black background, the date and the hour**.
+
+## I. **Action panel**
+
+**Bellow the panel’s header**, we have all the order's action button:
+
+-   **Update the order status**
+-   **Print order button**
+-   **View invoice show button**
+-   **View delivery slip button**
+-   **Partial refund button**
+-   **Standard Refund* button*
+-   **Return products button**
+-   **Cancel products button**
+-   **Cancel button**
+
+1.  **Update the order status**
+
+The current status is displayed by default on the droplist. It can be changed by selecting the new order's status from the droplist after validating the confirmation button.
+
+2.  **Print order button**
+
+Opens the Print page of the browser with the order’s content.
+
+[pdf file result 1/2](https://invis.io/YKTGGFZAMCB#/386350135_Order_Details_-_Print_Order_1)
+
+[pdf file result 2/2](https://invis.io/YKTGGFZAMCB#/386350136_Order_Details_-_Print_Order_2)
+
+3.  **View invoice show**
+
+If the current or new order status as **the generation of the invoice activated,**  **the button is shown or** the user generates it manually.
+
+When the user clicks on the button, it downloads the invoice.
+
+4.  **View delivery slip**
+
+If the current or new order status as **the generation of the delivery slip** then **the button to download the delivery slip is shown.**
+
+5.  **Partial refund button**
+
+If the current or new order status of order considered **as paid** then **the button is shown.** After clicking on the button, then the product table will take the full width of the page. The user can edit the refund amount by product and also the shipping cost. He must select the quantity of the product concerned for the refunded and then write refund amount chosen by product rows. The maximum refundable amount is written bellow all the amount input in tax included.
```

Comment:
"The maximum refundable amount is written bellow all the amount input in tax included" : Should this maximal refundable amount be checked when validating the partial refund ? Or is this just "for information" for the merchant ? (In Legacy it was just "For information")